.github/workflows/hadolint.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: hadolint
+on: [push, pull_request]
+jobs:
+  hadolint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: brpaz/hadolint-action@v1.2.1
+        with:
+          dockerfile: './container/webui/Dockerfile'
+      - uses: brpaz/hadolint-action@v1.2.1
+        with:
+          dockerfile: './container/worker/Dockerfile'
```

Comment:
I guess using our own shell style call for this could help to catch all Dockerfiles, not just the ones that are explicitly specified

Reply:
There is [#3](https://github.com/brpaz/hadolint-action/issues/3#issuecomment-646366920) about this, so basically we could run `hadolint ./container/**/Dockerfile` with `container: hadolint/hadolint`. Upstream seems happy to take fixes, not sure how easy that is.

Reply:
Yes, but what I am suggesting is to not rely on github action specific stuff that people can not reproduce locally and also is not complete but rather use simple shell scripts that solve both points, same as in https://github.com/os-autoinst/os-autoinst/pull/1604/files#diff-1af57dc1e7a5a58067566f2787f0030953af77313ae8d5488b06f71a851640b7R11

Reply:
That's a totally different point. But yes, I also brought it up in a daily last week, where these points came up:

- You can see the reproducer in the PR description but by default GHAs don't advertise how to reproduce them locally.
- GHAs are nice and concise snippets which we don't have to maintain.
- Usually makefile targets are the most convenient to reproduce something locally.

So maybe what we want here is a `test-containers-hadolint` target, and `test-containers-compose` for #3758 respectively.

Reply:
Well, not as straightforward as I thought, I'm getting all kinds of funny errors trying to simply translate the glob to something running in the container... but added the new target with two individual lines anyway

Reply:
Have seen and understood the line
https://github.com/os-autoinst/os-autoinst/pull/1604/files#diff-1af57dc1e7a5a58067566f2787f0030953af77313ae8d5488b06f71a851640b7R11
?

Maybe you forgot to pass the current working directory into the container because the globbing works on passing paths to hadolint within the container so hadolint needs to be able to find the Dockerfile's *within* the container environment

Reply:
If this is more obvious to you, I'd welcome a one-line suggestion to make it work in the `Makefile`.

Reply:
Variants including `-v $(pwd):/root --workdir=/root` and `$(readlink -f "$(dirname "$0")"/..):/root --workdir=/root`

Reply:
Well, I consider the best approach doing the same as in os-autoinst, so trying with #3772